### DISCUSSION TITLE
Phase unwarping failure

### DISCUSSION THREAD

**Discussion by carlesfalcon** [kschan0214/sepia#88] [Q&A]
Dear experts,
I have been asked to quantify a QSM study with SEPIA and I found an unexpected phase-unwarpping output (attached original and unwarped phase images). Looking at the paper with the ISMRM expert group recommendations (the dataset was previous to its publication), the problem seems to be related to the acquisition settings and/or parameters (coil combination-calibration). My question is if there is any way to overcome this issue. Otherwise, how much reliable would be the maps obviating a perfect phase unwarping (by restricting/masking the FOV to unaffected areas)?
Many thanks
Carles Falcon
University of Barcelona
![original_phase_image](https://github.com/kschan0214/sepia/assets/6885062/88fb275f-1944-4d41-be5e-6f45152717ea)
![unwarped_phase](https://github.com/kschan0214/sepia/assets/6885062/ddb8c612-d03c-4f20-93f4-36830d316ca5)

**Comment by kschan0214** [comment]
Hi Carles,

I think the original data suffered from the phase singularity issue which is indeed due to the suboptimal coil combination method. The issue is caused by destructive interference at certain locations where the phase information is lost after coil combination (which is usually the final output images from scanners). For the data that were already acquired, I don't think currently there is any reliable way to recover the lost information except having the uncombined coil images (or k-space data, which is not typically done). For future data acquisition, you may find the supporting information of the recommendation paper relevant https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fmrm.30006&file=mrm30006-sup-0003-Supinfo3.pdf

Kwok
